### PR TITLE
More fixes to rkt support

### PR DESCRIPTION
**Pull request by runningwild**
Just a couple minor changes.  Still working towards getting rkt working on kubernetes.

### CODE REVIEW

**Comment by runningwild** [comment]
Rkt support still isn't there.  Next step, as far as I can tell, is getting etcd running on coreos.  I think it expects it to be running by default but we're not explicitly starting it.  I know little about etcd, though, so I'm not sure how to set it up, I assume k8s wants it set up in some specific way.  If someone could point me towards some information on how k8s wants etcd configured I can start working on that.

**Comment by yujuhong** [comment]
@runningwild, what etcd are you talking about? The etcd on the master node is started as a pod by kubelet. 

/cc @dchen1107, @yifan-gu

**Comment by runningwild** [comment]
I'm talking about the minions.  At least from reading the coreos's journal it looks like something is expecting etcd to be running.  I didn't save the logs from the last time I tried this though so I don't remember what exactly it is that needs it.

**Comment by yifan-gu** [comment]
I am also confused why etcd is required?

File: cluster/gce/coreos/node.yaml [hunk 1]
```diff
@@ -93,6 +93,7 @@ coreos:
         EnvironmentFile=/etc/kube-env
         ExecStartPre=/usr/bin/rm -rf /opt/rkt
         ExecStartPre=/usr/bin/mkdir -p /opt/rkt
+        ExecStartPre=/usr/bin/mkdir -p /etc/rkt
```
Comment (yifan-gu) [inline cluster/gce/coreos/node.yaml]: what's this for? can't use /opt/rkt?

Comment (runningwild) [inline cluster/gce/coreos/node.yaml]: rkt uses /etc/rkt as its local config directory.  So it either needs to exist, or we can change the directory it uses by specifying --local-config=/something/else on every rkt call.

**Comment by runningwild** [comment]
The minions need etcd, right?  I'm not sure the specifics of what data is shared with the minions via etcd, but it runs in a pod with several other containers and it always dies because of etcd.  I found the problem, though - the etcd docker image doesn't specify an entrypoint, so rkt gives up on it pretty quickly.  I made an etcd docker image on top of the one in gcr.io and specified the entry point and flags and it works fine.

File: cluster/gce/coreos/node.yaml (hunk 1 above)
Comment (yifan-gu) [inline cluster/gce/coreos/node.yaml]: @runningwild Cool, forgot that :stuck_out_tongue:

**Comment by yujuhong** [comment]
> The minions need etcd, right? I'm not sure the specifics of what data is shared with the minions via etcd, but it runs in a pod with several other containers and it always dies because of etcd. I found the problem, though - the etcd docker image doesn't specify an entrypoint, so rkt gives up on it pretty quickly. I made an etcd docker image on top of the one in gcr.io and specified the entry point and flags and it works fine.

@runningwild, iiuc, it should be the etcd on the master node, which is the only etcd instance kubernete starts.

This PR LGTM. Thanks!

**Pull request merged**